src/main/java/com/example/springboot/model/ShoppingCart.java

```diff
@@ -0,0 +1,27 @@
+package com.example.springboot.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
```

Comment:
it's better to ident annotated fields
```suggestion
    @Id
    private Long id;
    
    @OneToOne
    @JoinColumn(name = "id", nullable = false)
    private User user;
    
    @OneToMany(mappedBy = "shoppingCart")
    private Set<CartItem> cartItems = new HashSet<>();
```